src/handlers/transfers/index.ts

```diff
@@ -134,66 +135,44 @@ export const handlers: CallHandler[] = [
       });
     },
   },
-  // ERC20 "transferFrom"
+  // Unified ERC20/ERC721 transferFrom with heuristic
   {
     selector: iface.getSighash("transferFrom"),
     handle: (state: StateChange, payments: Payment[], trace: CallTrace) => {
-      // The way to differentiate ERC20 from ERC721 "transferFrom"
-      // is by checking the return value (which is a boolean value
-      // for ERC20 and is missing for ERC721)
-      if (trace.output && trace.output !== "0x") {
```

Comment:
To simplify the changes, can we keep the logic as it is but instead just change the condition:
`(trace.output && trace.output !== "0x") || knownNonStandardERC20.includes(trace.to)`